Game_Manual/Stats.md

```diff
@@ -23,40 +23,97 @@ The average % quality of all projects you've worked on.
+How much your team members felt you contributed positively to the team culture. This is a weighted average with more recent projects counting more than earlier ones. Represented as a percentage (0..100%).
 
-## Discernment
+## Project Contribution
 
-Reflects the degree to which you are biased in your assessment of your contribution to projects (as compared to the average of others' assessment). Represented as a percentage corresponding to the average of how much you over- or under-estimate your contribution to projects.
+The amount that you contribute to a project determines a series of other stats, and also contains several sub-stats within.
 
-Zero is equal to perfect discernment (i.e. you estimate your contribution the same as your peers do). The closer this stat is to zero, the more accurate your discernment is.
+Within a given project, you have **actual** and **expected** contributions, as well as a **contribution gap**.
 
-#### Example
+To demonstrate how the various contribution stats are calculated, we'll use the following dataset in examples.
 
-Let's say you worked on three projects, with the following contribution rankings:
+#### Example Project Contribution Estimations
 
-| Project      | Contribution (self-estimated) | Contribution (team-estimated) |
-|:-------------|:------------------------------|:------------------------------|
-| #tiny-tigers | 20%                           | 30%                           |
-| #big-bees    | 50%                           | 45%                           |
-| #red-rabbits | 28%                           | 30%                           |
+| Project      | Team size | Self estimate | Team estimates |
+|:-------------|:----------|:--------------|:---------------|
+| #big-bees    | 3         | 50%           | 42%, 48%       |
+| #red-rabbits | 4         | 28%           | 27%, 35%, 28%  |
+| #tiny-tigers | 2         | 20%           | 30%            |
 
-Your discernment _per project_ is the difference between self- and team-estimated contribution as an absolute number. So the discernment stats for the above projects would be:
+### Actual Contribution
 
-| Project      | Discernment          |
+Your actual contribution is the average of your team's estimates of what percentage of the project you contributed to. It includes your own estimate of your contribution. Represented as a percentage (0..100%).
+
+In the above dataset, the player's actual contribution for the `#big-bees` project is the average of _all_ the estimates, so it would be equal to 46.67% ( (50% + 42% + 48%) / 3 ).
+
+### Expected Contribution
+
+Your expected contribution how much you are expected to contribute to the project based on the team size so that each player contributes equally. Represented as a percentage (0..100%).
+
+In the above dataset, the player's expected contribution for the `#big-bees` project is equal to 33.33% ( 100% / 3 ) because their team has 3 players, so an equal contribution is 1/3 of the project.
+
```

Comment:
Ok, I wasn't aware of this. Hasn't been a part of the way I've been calculating expected contribution, so we've got some communication dissonance here. See my comment above: https://github.com/LearnersGuild/playbook/pull/41#discussion_r74252554

I'll update the description.